.github/workflows/tests.yml

```diff
@@ -58,8 +59,8 @@ jobs:
             python: 3.9
             geos: 3.10.3
             numpy: 1.19.5
-          # pypy
-          - os: ubuntu-latest
+          # pypy (don't use ubuntu-latest to not overwrite existing geos 3.11.0 build)
```

Comment:
nit: double negative, suggest rephrasing to "Pin ubuntu version to ensure GEOS 3.11.0 build"

Reply:
Thanks, reworded a bit to remove the double negative!